test/kubemark/start-kubemark.sh

```diff
@@ -16,6 +16,7 @@
 
 # Script that creates a Kubemark cluster for any given cloud provider.
 
+set -x
```

Comment:
please avoid set -x, as some password may be shown in logs

Reply:
fixed, was in some deep debugging